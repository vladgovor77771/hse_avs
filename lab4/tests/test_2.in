4
qxyz
2
01234567890123456789012345678901234567890123456789012345678901
ouyt
2
01234567890123456789012345678901234567890123456789012345678901
mnkj
2
01234567890123456789012345678901234567890123456789012345678901
abcd
2
01234567890123456789012345678901234567890123456789012345678901
