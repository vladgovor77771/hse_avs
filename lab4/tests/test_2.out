abcd => 0123
mnkj => 2309
ouyt => 4049
qxyz => 6345
